ca/README.md

```diff
@@ -0,0 +1,49 @@
+# CFSSL Certificate Authority
+
+This repo is used to run a Certificate Authority with CFSSL
+
+CFSSL binaries are installed. On startup the container will run these scripts.
+
+1. 60-cfssl: start the CFSSL certificate authority server.
+2. 90-cfssl-gencert: Generate container's own certificate and try to remote sign it with the CA.
+3. 91-cfssl-cacert: Download and install the CA public cert from the CA.
+
+It can be controlled via the following variables, some of them inherited from base container.
+
+`OPG_CA_CLIENTAPIKEY` - Hex string for the password to use the client signing profile
+
+`OPG_CA_SERVERAPIKEY` - Hex string for the password to use the server signing profile
+
+`OPG_CA_CLIENTNAMEWHITELIST` - Regex string to limit the domains that will be signed for the client signing profile. Backslashes must be escaped for use in the json template.
+
+`OPG_CA_CSR_CN`, `OPG_BASE_CSR_CN` - The `common name` attribute of the certificate.
```

Comment:
Do we want to have both these certs on th CA? We could remove the base container CFSSL SSL cert gen or reuse `SKIP_SSL_GENERATE`